source/client/service_impl.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+#ifdef __clang__
+#pragma clang diagnostic push
+#pragma clang diagnostic warning "-Wunused-parameter"
+#endif
+#include "api/client/service.grpc.pb.h"
+
+#ifdef __clang__
+#pragma clang diagnostic pop
+#endif
```

Comment:
Maybe just set `-Wunused-parameter` in the Bazel target `copts`..

Reply:
Let me figure out why I tackled this like I did here, and ping you on Slack when I did that to discuss. I think this was CI specific.

Reply:
As discussed out of band, `grpc_cc_proto` doesn't allow us to pass in `copts`, so we'd have to apply the suppression to a wider scope; I propose we keep this as-is.